web/src/Layout.tsx

```diff
@@ -182,6 +185,7 @@ export const Layout: React.FunctionComponent<LayoutProps> = props => {
             {!isSiteInit && !isSignInOrUp && (
                 <GlobalNavbar
                     {...props}
+                    authRequired={Boolean(authRequired)}
```

Comment:
```suggestion
                    authRequired={!!authRequired}
```

`!!` is more idiomatic and safer than using the `Boolean` function